src/components/common/FormField.js

```diff
@@ -80,7 +80,7 @@ class FormField extends Component {
           value={this.props.value}
           keyboardType={this.props.keyboardType}
           maxLength={this.props.maxLength}
-          autoCorrect={this.props.autoCorrect}
+          autoCorrect={this.props.autoCorrect || false}
           autoFocus={this.state.autoFocus}
```

Comment:
These have been inserted to remove any potential vulnerability whereby users' private key input could be sent to Google or Apple